### PR TITLE
more explicitly say privacy is maintained in Security Considerations

### PR DESCRIPTION
**Pull request by danwing**

### CODE REVIEW

File: draft-wing-cidfi.md [hunk 1]
```diff
@@ -1009,11 +1009,10 @@ the packet importance.  This was a design trade-off to reduce the
 CPU effort on the CNEs.  A mitigation is using
 several DCIDs for every packet importance.
 
-Communications are relayed through the client because only the
+The server's identity is not disclosed to the CIDFI Network Elements,
+thus maintaining the end user's privacy.  Communications are relayed through the client because only the
 client and server knows the identity of the server and can validate
```
Comment (boucadair) [inline draft-wing-cidfi.md:1014]:
```suggestion
client knows the identity of the server and can validate
```

File: draft-wing-cidfi.md [hunk 2]
```diff
@@ -1009,11 +1009,10 @@ the packet importance.  This was a design trade-off to reduce the
 CPU effort on the CNEs.  A mitigation is using
 several DCIDs for every packet importance.
 
-Communications are relayed through the client because only the
+The server's identity is not disclosed to the CIDFI Network Elements,
```
Comment (boucadair) [inline draft-wing-cidfi.md:1012]:
```suggestion
Other than what can be inferred from a destination IP address, the server's identity is not disclosed to the CIDFI Network Elements,
```

**Review by boucadair — CHANGES_REQUESTED** [review]
No more data revealed other than what can be (already) inferred from a destination IP address.

**Review by danwing — COMMENTED** [review]
yep.

**Pull request merged**